lib/app/layouts/conversation_view/widgets/message/popup/message_popup.dart

```diff
@@ -981,11 +969,11 @@ class _MessagePopupState extends OptimizedState<MessagePopup> with SingleTickerP
 
   void unsend() async {
     popDetails();
-    final response = await http.unsend(message.guid!, partIndex: part.part);
-    if (response.statusCode == 200) {
-      final updatedMessage = Message.fromMap(response.data['data']);
-      ah.handleUpdatedMessage(chat, updatedMessage, null);
+    final updatedMessage = await backend.unsend(message, part);
+    if (updatedMessage == null) {
+      return;
     }
+    ah.handleUpdatedMessage(chat, updatedMessage, null);
```

Comment:
probably makes more sense to have the opposite condition rather than returning and continuing